packages/eslint-plugin-pf-codemods/src/rules/helpers/renameInterface.ts

```diff
@@ -0,0 +1,84 @@
+import { Rule } from "eslint";
+import { TSESTree } from "@typescript-eslint/utils";
+import { Identifier, ImportSpecifier, Node } from "estree-jsx";
+import { getFromPackage } from "./getFromPackage";
+
+interface InterfaceRenames {
+  [currentName: string]: string;
+}
+
+function formatDefaultMessage(oldName: string, newName: string) {
+  return `${oldName} has been renamed to ${newName}.`;
+}
```

Comment:
Not a blocker, but wondering if we should add a directory to `helpers` for these sort of default message functions. `defaultMessages` as the directory -> `renameIdentifier.tsx` as this helper for example. Or just yanking this out as its own individual helper to be reused elsewhere.